### ISSUE TITLE
Impress fails to start

### ISSUE THREAD

**Issue by belochub** [metarhia/impress#771]
Impress version 0.4.25
Node.js version 6.11.2
Stacktrace:
```
TypeError: Cannot read property 'replace' of undefined
    at impress.STACK_REGEXP.forEach (/home/metarhia/server/node_modules/impress/lib/impress.js:58:18)
    at Array.forEach (native)
    at EventEmitter.impress.normalizeStack (/home/metarhia/server/node_modules/impress/lib/impress.js:57:24)
    at EventEmitter.impress.fatalError (/home/metarhia/server/node_modules/impress/lib/impress.js:81:19)
    at EventEmitter.impress.serverOnError (/home/metarhia/server/node_modules/impress/lib/impress.js:542:15)
    at Server.listener.on (/home/metarhia/server/node_modules/impress/lib/impress.js:550:13)
    at emitTwo (events.js:106:13)
    at Server.emit (events.js:191:7)
    at TcpServer.<anonymous> (/home/metarhia/server/node_modules/metarhia-jstp/lib/common.js:57:8)
    at emitTwo (events.js:106:13)
    at TcpServer.emit (events.js:191:7)
    at Server.<anonymous> (/home/metarhia/server/node_modules/metarhia-jstp/lib/common.js:57:8)
    at emitOne (events.js:96:13)
    at Server.emit (events.js:188:7)
    at emitErrorNT (net.js:1277:8)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```
To get the same stacktrace you should add line
```javascript
console.log(err);
```
on line 104 of `lib/impress.js`.

**Comment by tshemsedinov** [comment]
Fixed in impress 0.4.26

**Comment by tshemsedinov** [comment]
Fix commit #772
